src/contracts/Hub.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Eth } from 'web3-eth';
+import { HttpProvider } from 'web3-providers';
+import { Environment } from '../Environment';
+import deployment from '../deployments/mainnet';
+import { Hub, FundRoutes } from './Hub';
+import { toChecksumAddress } from 'web3-utils';
+
+describe('Hub', () => {
+  let environment: Environment;
+  let hub: Hub;
+  let routes: FundRoutes;
```

Comment:
You don't need to create a shared `routes` variable up here unless you want to share it across different tests.

Reply:
Actually sharing it between two tests.

Reply:
Alright then